### PR TITLE
Shopify config

### PR DESCRIPTION
**Pull request by jcech**
Initial copy of the Shopify `rubocop.yml`. I did exclude vendor/ruby/** in case your gems are bundled in `vendor`. 

To test it out on a project add/modify` gem 'argon-rubocop', git: 'https://github.com/planetargon/argon-rubocop', require: false, branch: 'shopify-config'` to the Gemfile.

### CODE REVIEW

**Comment by jguest** [comment]
@jcech looks good to me

**Review by jguest — APPROVED** [review]

File: config/rubocop.rails.yml [hunk 1]
```diff
@@ -1,37 +1,1179 @@
...
+Lint/BigDecimalNew:
+  Enabled: true
+
+Style/TrailingBodyOnClass:
+  Enabled: true
+
+Style/TrailingBodyOnModule:
+  Enabled: true
+
+Style/TrailingCommaInArrayLiteral:
```
Comment (dudleysr) [inline config/rubocop.rails.yml:1162]: Not sure how I feel about Trailing Commas, even in Multiline Hashes and Arrays. I know it helps with diffs but it just looks...weird to me. Thoughts?

File: config/rubocop.rails.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 AllCops:
   Exclude:
   - 'db/schema.rb'
+  - 'vendor/ruby/**/*'
```
Comment (dudleysr) [inline config/rubocop.rails.yml]: Thoughts on adding `node_modules`, `bin/yarn`, or other common JS related directories to Exclude? Or keep this as something that should be added in the local `.rubocop.yml`?

**Review by dudleysr — COMMENTED** [review]
I finally figured out how to merge Excludes and other methods so I don't think adding the JS directories is relevant now.

The Trailing Commas, though, I think is worth discussion

**Comment by katcarr** [comment]
👍good to merge

**Pull request merged**